src/openforms/fixtures/default_admin_index.json

```diff
@@ -205,5 +214,38 @@
         "name": "Test email backend",
         "link": "/admin/email/test/"
     }
+},
+{
+    "model": "admin_index.applink",
+    "fields": {
+        "order": 1,
+        "app_group": [
+            "handige-links"
+        ],
+        "name": "API Documentatie",
+        "link": "/api/docs"
```

Comment:
For me this doesn't work without having an extra slash (`/api/docs/`)

Reply:
Updated the fixture, apparently it's because of the capture-all nature of `openforms.forms.urls` :(